api-mobile/src/queries/activity-queries.ts

```diff
@@ -29,7 +29,7 @@ export const postActivitySQL = (activityData: ActivityPostBody): ParameterizedQu
       $2,
       $3,
       $4,
-      ST_Force2D(
+      public.ST_Force2D(
```

Comment:
we removed the geom from the input values as well so there's a line of code above this to re-add it to the query as well as uncomment the geom object itself and additionally add it back to the array below before this can be merged

Reply:
disabled it for the demo

Reply:
Great. Thanks.
I've got a demo ready to go that show the geom being inserted.

Reply:
Once your changes are made to this PR let me know and we can see if the polygon flows through the iOS app to the db, would be cool to see if we can figure out some GeoJson queries to show the polygons in metabase.